.github/workflows/check-setup-py.yml

```diff
@@ -87,8 +87,8 @@ jobs:
       - name: install ibis in development mode
         run: pip install -e .
 
-      - name: run simple tests
-        run: pytest ibis/tests --numprocesses auto
+      - name: run tests
+        run: pytest -m 'not backend' --numprocesses auto
```

Comment:
I wonder, could we mark `not backend` tests implicitly with its own marker, so we could run `pytest -m ibis` or `pytest -m core` instead of `pytest -m 'not backend'`?